app/src/main/java/com/safetyheads/akademiaandroida/PasswordValidator.kt

```diff
@@ -20,15 +20,15 @@ class PasswordValidator {
         }
     }
 
-    private fun validatePassword(password: String, editText: EditText, context: Context) {
-        if (!isValid(password)) {
-            editText.error = context.getString(R.string.invalid_password)
-        } else {
-            editText.error = null
-        }
-    }
+     fun validatePassword(password: String, editText: EditText, context: Context) {
+         if (!isValid(password, editText, context)) {
+             editText.error = context.getString(R.string.invalid_password)
+         } else {
+             editText.error = null
+         }
+     }
 
-    private fun isValid(password: String): Boolean {
+    fun isValid(password: String, editText: EditText, context: Context): Boolean {
```

Comment:
Why you change method signature? EditText and Context is not used, please remove it.